### DISCUSSION TITLE
Problems with plugins after upgrading

### DISCUSSION THREAD

**Discussion by tester22** [drakkan/sftpgo#1783] [General]
I am using the SFTPGo marketplace offering on Azure.
After upgrading SFTPGo from 2.5.6 to 2.6.2 using the built in package manager I am getting the following error if any plugin is enabled:

unable to create events searcher plugin: Unrecognized remote plugin message: NAME:
This usually means
  the plugin was not compiled for this architecture,
  the plugin is missing dynamic-link libraries necessary to run,
  the plugin is not executable by this process due to file permissions, or
  the plugin failed to negotiate the initial go-plugin protocol handshake
  
Additional notes about plugin:
  Path: /srv/sftpgo/plugins/sftpgo-plugin-eventsearch-linux-amd64
  Mode: -rwxrwxrwx
  Owner: 998 [sftpgo] (current: 998 [sftpgo])
  Group: 999 [sftpgo] (current: 999 [sftpgo])
  ELF architecture: EM_X86_64 (current architecture: amd64)

Without plugins SFTPGo is working as expected.

**Comment by drakkan** [comment]
Hi @tester22, as commercial user, please contact us using our dedicated email address (support at sftpgo dot com) providing your Azure Subscription ID so we can validate your installation. Thank you!
